UPGRADING.md

```diff
@@ -32,6 +32,52 @@ clientCtx = clientCtx.
 
 Refer to SimApp `root_v2.go` and `root.go` for an example with an app v2 and a legacy app.
 
+#### Textual sign mode
```

Comment:
Can we add this here https://github.com/cosmos/cosmos-sdk/pull/18166#pullrequestreview-1687123506 as well, as Marko said?

Reply:
added the link to the docs section and the ADR. I've also opened a PR on the docs repo

Reply:
the advanced and learn section is still tracked from the sdk, so the changes should be here.
the docs repo manages the building of the docs and probably later tutorials.